frontend/src/lib/components/InsightLabel/index.tsx

```diff
@@ -34,10 +35,8 @@ interface InsightsLabelProps {
     hasMultipleSeries?: boolean // Whether the graph has multiple discrete series (not breakdown values)
     showCountedByTag?: boolean // Force 'counted by' tag to show (always shown when action.math is set)
     allowWrap?: boolean // Allow wrapping to multiple lines (useful for long values like URLs)
-    useCustomName?: boolean // Whether to show new custom name. `{custom_name} ({id})`.
-    hideSeriesSubtitle?: boolean // Whether to show the base event/action name (if a custom name is set) in the insight label
+    hideCustomName?: boolean // Whether to show the base event/action name (if a custom name is set) in the insight label
```

Comment:
I might be missing it, but I don't think we ever set this to true. Maybe we can simplify by just removing it?